### ISSUE TITLE
Multiple member select, for job submission?

### ISSUE THREAD

**Issue by jellypuno** [zowe/vscode-extension-for-zowe#1548]
As a ZE User I would like to have the ability to submit multiple JCLs at the same time.

The process could be: 
1. Select multiple JCLs 
2. Right-click > Submit JCL
3. All selected JCLs will be submitted (in parallel?)

Out of scope:
- Submission of JCL using different profiles 

Currently, ZE displays an error message if you submit a PDS member that is not a JCL. This error message should be displayed as well if ever users selected a combination of JCL and non JCL members

**Comment by jellypuno** [comment]
Different Issue

Research how we could optimize our call for multi-select functionality.
- Find a way how to not duplicate code.

**Comment by crawr** [comment]
> Different Issue
> 
> Research how we could optimize our call for multi-select functionality.
> 
> * Find a way how to not duplicate code.

In the new research issue, we can consider this suggestion as well: https://github.com/zowe/vscode-extension-for-zowe/pull/1583/files#r756801485
